### PR TITLE
Replace /tta-service path with the /teacher-training-adviser/sign_up/identity

### PR DESCRIPTION
**Pull request by plisovin**
### Trello card

[Trello-4692](https://trello.com/c/luf91eZ2/4692-replace-the-tta-service-redirect-to-the-adviser-funnel-post-migration)

### Context

We currently have a /tta-service redirect in place on GIT. This was created when the adviser funnel was on a separate site, so that links to the adviser site would always point to the right environment (eg links from the dev environment of GIT would always open the dev environment of the adviser site).

Now that the adviser funnel has been migrated to GIT, we can remove this redirect as it is not good for SEO.

### Changes proposed in this pull request

Replaced all the redirect links to the adviser funnel with normal internal links
Removed the /tta-service redirect from the redirects config file

### CODE REVIEW

File: app/views/content/blog/9-tips-to-help-you-apply-for-teacher-training.md
Comment (EllieNodder) [inline app/views/content/blog/9-tips-to-help-you-apply-for-teacher-training.md:1]: @gemmadallmandfe @philspray I know this ticket is just about removing the /tta-service redirect, but do we have a wider question now of when we should be linking straight to the start of the GIT funnel, and when we should be linking to the /teacher-training-advisers page? (not sure if we already have a ticket on this?)

On this blog alone, we have 2 links to the content page, and 1 straight to the funnel, which seems a bit odd, a) to have quite so many links to advisers, and b) to have links to different _places_ for advisers

Comment (gemmadallmandfe) [reply app/views/content/blog/9-tips-to-help-you-apply-for-teacher-training.md:1]: Good shout @EllieNodder !

I'll raise a separate ticket for this in the backlog - I think for now we should just concentrate on getting this redirect replaced?

**Comment by gemmadallmandfe** [comment]
@plisovin These changes all look good to me

However, there are other references to `/tta-service` in GitHub:

* ‎.env.test
* ‎config/routes.rb
* lib/disable_turbolinks.rb
* spec/features/content_pages_spec.rb
* ‎spec/lib/disable_turbolinks_spec.rb
* ‎spec/requests/pages_controller_spec.rb

I'm presuming these will all need clearing up as well - but presumably we could do this under a separate tech debt ticket?

If you agree, I'll tick this PR so it can be merged tomorrow (Thursday) and will then raise a separate tech debt ticket for when you return from leave.

**Comment by plisovin** [comment]
> @plisovin These changes all look good to me
> 
> However, there are other references to `/tta-service` in GitHub:
> 
> * ‎.env.test
> * ‎config/routes.rb
> * lib/disable_turbolinks.rb
> * spec/features/content_pages_spec.rb
> * ‎spec/lib/disable_turbolinks_spec.rb
> * ‎spec/requests/pages_controller_spec.rb
> 
> I'm presuming these will all need clearing up as well - but presumably we could do this under a separate tech debt ticket?
> 
> If you agree, I'll tick this PR so it can be merged tomorrow (Thursday) and will then raise a separate tech debt ticket for when you return from leave.

I think it's to do it here. Pushed an additional changes

**Review by asatwal — DISMISSED** [review]
👍 LGTM

**Comment by gemmadallmandfe** [comment]
Looks good to me

One question/suggestion - I think we should probably add a new redirect as follows, just in case there are any referral links (there shouldn't be but it wouldn't hurt to cover that eventuality?):

- FROM `/tta-service`
- TO `/teacher-training-adviser/sign_up/identity`

Other than that, very happy for you to merge first thing on Friday morning! We should then monitor and just check that adviser sign-ups are continuing as normal

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**